docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+
+The link:https://registry.terraform.io/providers/redhat-developer/rhoas/latest[RHOAS Terraform^] provider is available in the official link:https://www.terraform.io/[Terraform provider registry^] and includes resources to interact with {product-long-rhoas}.
+
+You can fully manage your Kafka environment through your Terraform system using the RHOAS Terraform provider. You can create, retrieve info, configure, deploy, and delete the following {product-kafka} resources:
+
+* Kafka instances
+* Service accounts
+* Kafka topics
+* Access Control Lists (ACLs)
+
+
+[id="proc-using-rhoas-terraform-provider_{context}"]
+=== Installing the RHOAS Terraform provider
+
+.Prerequisites
+* You have a Red Hat account.
+* https://www.terraform.io/downloads[Terraform^] v1.3.4 or later is installed.
+* You have an offline token that is accessible to Terraform.
+
+[NOTE]
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and can be found at https://cloud.redhat.com/openshift/token. As the offline token is a sensitive value that varies between environments it is best specified through as `_OFFLINE_TOKEN_` environment variable when running `terraform apply` in a terminal.
+
+.Procedure
+. Open your browser of choice and navigate to the RHOAS Terraform provider.
+. Click *Use Provider*.
+. Copy the code provided.
++
+
+.Example rhoas provider configuration
+[source,shell]
+----
+$ terraform {
+  required_providers {
+    rhoas = {
+      source = "redhat-developer/rhoas"
+      version = "0.3.0"
+    }
+  }
+}
+
+provider "rhoas" {
+  #configuration options
+}
+----
+
+. Open your IDE of choice and paste the code into it. You can specify configuration options in the provider settings. For example, you can enter the offline token here using the `offline_token` field. However as noted previously, it is recommended to enter the offline token as an environment variable.
++
+
+. Save the file as a .tf file.
+. Open a terminal and enter the following command in the Terraform directory. This command initializes the working directory containing Terraform configuration files and installs any required plug-ins.
++
+[source,shell]
+----
+$ terraform init
+----
+A success message displays that `Terraform has been successfully initialized!`
+
+[id="con-using-terraform_{context}"]
+=== Using the RHOAS Terraform provider
+
+Resources are the most important element in the Terraform language. Each resource block describes one or more infrastructure objects. Terraform has error messages that help with minor typing errors by suggesting a resource close to what is typed. The next steps show what resources (either for single tasks or a combination of tasks) you can add to your Terraform configuration file to provision your Kafka instance.
```

Comment:
Similar to https://github.com/redhat-developer/app-services-guides/pull/601/files#r1027975390, we should tie this Terraform vocabulary back to OpenShift Streams. If I understand it correctly, it might be something like the following:

```
.  Resources are the most important element in the Terraform language. Each resource block in a Terraform provider describes one or more infrastructure objects. For {product-long-kafka}, such infrastructure objects might include Kafka instances, service accounts, Access Control Lists (ACLs), and topics.
```

Reply:
I think I'd remove the sentence beginning "Terraform has error messages...."

Reply:
Suggest something like this: "The procedures that follow show what resources you can add to your Terraform configuration file to create a Kafka instance and its associated resources such as service accounts and topics."